interface/login/login.php

```diff
@@ -35,6 +35,8 @@
 <html>
 <head>
 <?php html_header_show();?>
+<TITLE><?php xl ('OpenEMR Login','e'); ?></TITLE>
```

Comment:
Actually, likely makes more sense to do:
``````
<TITLE><?php echo text($openemr_name) . " " . xlt('Login'); ?></TITLE>
``````